### PR TITLE
ENT-8810 - 'Improved node diagnostics' release note update for 4.10, text approved by Adel

### PR DESCRIPTION
**Pull request by paulmoloneyr3**

### CODE REVIEW

File: content/en/platform/corda/4.10/community/release-notes.md [hunk 1]
```diff
@@ -51,6 +51,13 @@ Postgres 13.8 is now supported.
 
 The OpenTelemetry tracing signal is now supported in flows across nodes. For more information, see [OpenTelemetry](opentelemetry.md).
 
+### Improved node diagnostics
+
+This release includes improved node diagnostics:
+* Every five minutes, there is a thread dump to the log file
+* There is a periodic check to determine if the state machine thread pool seems to be blocked, and a warning is generated if so.
```
Comment (colin-hillis) [inline content/en/platform/corda/4.10/community/release-notes.md:58]:
```suggestion
* There is a periodic check to determine if the state machine thread pool is blocked and a warning is generated if so.
```

**Review by nadinequinr3 — APPROVED** [review]

File: content/en/platform/corda/4.10/community/release-notes.md [hunk 2]
```diff
@@ -51,6 +51,13 @@ Postgres 13.8 is now supported.
 
 The OpenTelemetry tracing signal is now supported in flows across nodes. For more information, see [OpenTelemetry](opentelemetry.md).
 
+### Improved node diagnostics
+
+This release includes improved node diagnostics:
+* Every five minutes, there is a thread dump to the log file
```
Comment (colin-hillis) [inline content/en/platform/corda/4.10/community/release-notes.md:57]:
```suggestion
* There is a thread dump to the log file every five minutes.
```

File: content/en/platform/corda/4.10/community/release-notes.md [hunk 3]
```diff
@@ -51,6 +51,13 @@ Postgres 13.8 is now supported.
...
+* There is a periodic check to determine if the state machine thread pool seems to be blocked, and a warning is generated if so.
+* Log messages are now output, on both nodes initiating flows on other flows and on the receiving nodes, to tie outgoing initiate sessions to their message ID and likewise for inbound initiate sessions to their message ID so logs across nodes can be more easily diagnosed.
```
Comment (colin-hillis) [inline content/en/platform/corda/4.10/community/release-notes.md:59]:
```suggestion
* Log messages are now output, both on the nodes initiating flows on other flows and also on the receiving nodes. This ties outgoing initiate sessions to their message ID and also inbound initiate sessions to their message ID. This enables easier diagnosis of logs across nodes.
```

File: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md [hunk 4]
```diff
@@ -67,6 +67,13 @@ Postgres 13.8 is now supported.
 
 The OpenTelemetry tracing signal is now supported in flows across nodes. For more information, see [OpenTelemetry](node/operating/monitoring-and-logging/opentelemetry.md).
 
+### Improved node diagnostics
+
+This release includes improved node diagnostics:
+* Every five minutes, there is a thread dump to the log file
+* There is a periodic check to determine if the state machine thread pool seems to be blocked, and a warning is generated if so.
+* Log messages are now output, on both nodes initiating flows on other flows and on the receiving nodes, to tie outgoing initiate sessions to their message ID and likewise for inbound initiate sessions to their message ID so logs across nodes can be more easily diagnosed.
+
```
Comment (colin-hillis) [inline content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md:77]: Apply same changes to above 5 lines.

**Review by colin-hillis — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.10/community/release-notes.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/community/release-notes.md:58]: Done.

File: content/en/platform/corda/4.10/community/release-notes.md (hunk 2 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/community/release-notes.md:57]: Done.

File: content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md (hunk 4 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.10/enterprise/release-notes-enterprise.md:77]: Done.

**Review by colin-hillis — APPROVED** [review]
LGTM

**Pull request merged**